code/index.html

```diff
@@ -8,30 +8,47 @@
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
+    <header id="page-header"class="header-styling">Cluedo Mystery-picker</header>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+        <p id="killer-headline" class="killer-card-headline"></p>
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
         <img id="killerImage" src="" height="100" />
         <p id="killerDescription"></p>
+        <p id="killerFavouriteWeapon"></p>
+        <div id="loader-wrapper" class="load-wrapper">
+          <div id="load" class="loader">
+            <!--this is loader-->
+          </div>
+        </div>
       </article>
       <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
+        <p id="weapon-headline" class="weapon-card-headline"></p>
         <p id="weaponName"></p>
         <p id="weaponWeight"></p>
+        <div id="loader-wrapper" class="load-wrapper">
+          <div id="load2" class="loader">
+            <!--this is loader-->
+          </div>
+        </div>
       </article>
       <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
+        <p id="room-headline" class="room-card-headline"></p>
         <p id="roomName"></p>
```

Comment:
Clear namings overall on classes and ids, only thing you could think about is to be consistent in using camel/kebab case for readbility